astropy/units/quantity_helper/helpers.py

```diff
@@ -355,6 +355,12 @@ def helper_clip(f, unit1, unit2, unit3):
 # ufuncs that require dimensionless input and and give dimensionless output
 dimensionless_to_dimensionless_ufuncs = (np.exp, np.expm1, np.exp2, np.log,
                                          np.log10, np.log2, np.log1p)
+# Default numpy does not ship an "erf" ufunc, but some versions hacked by
+# intel do.  This is bad, since it means code written for that numpy will
+# not run on non-hacked numpy.  But still, we might as well support it.
+if hasattr(np, "erf"):
+    dimensionless_to_dimensionless_ufuncs += (np.erf,)
```

Comment:
This looks good to me, but the previous workaround was slightly different:

```
if isinstance(getattr(np.core.umath, 'erf', None), np.ufunc):
    dimensionless_to_dimensionless_ufuncs += (np.core.umath.erf,)
```

Reply:
That really is better. Let me put it back...

Reply:
Goes to show that my past self was cleverer...